### ISSUE TITLE
Add LICENSE.txt file

### ISSUE THREAD

**Issue by uvtc** [haxeui/haxeui-core#290]
Just noticed that in the CONTRIBUTING.md file you refer to the Apache 2 license and link to the license file, but there is no such file present.

Actually, looks like you need to add a LICENSE.txt file <http://www.apache.org/licenses/LICENSE-2.0.txt> to all of the haxeui project repos. I'm not seeing it in any of them.

There should also probably be a blurb at the very bottom of every project's README.md saying something like, "## License\n\nThis software is distributed under the terms of the Apache 2.0 license. See the included LICENSE.txt file for details.".

**Comment by dbriemann** [comment]
I agree. I was interested in the haxeui license as well but I only found this via Google.
